examples/examples-cpu/nyc-taxi-snowflake/xgboost-dask.ipynb

```diff
@@ -333,7 +331,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Convert to single-node DataFrames using `compute()` to calculate metrics with scikit-learn. If the data was larger, could use `dask_ml.metrics.mean_squared_error`"
+    "The metrics functions in `dask_ml` can compute metrics on Dask collection like Dask Array and Dask DataFrame, so you never have to hold all of the test data in memory on the client."
```

Comment:
```suggestion
    "The metrics functions in `dask_ml` can compute metrics on Dask collections like Dask Array and Dask DataFrame, so you never have to hold all of the test data in memory on the client. These functions intentionally mimic the metrics functions in scikit-learn.
```

Reply:
I did this manually since it needs to be done in two places:

https://github.com/saturncloud/examples/pull/41/commits/3f04808e017faee4bc3b1c25520d4f8442f868bf